### ISSUE TITLE
Backdoor config API doesn't work for HOSTNAME_EXTERNAL

### ISSUE THREAD

**Issue by sougat818** [localstack/localstack#3577]
# Type of request: 

[x] bug report

# Detailed description

Running local stack backdoor config api to override External Hostname for SQS doesn't reflect. Setting HOSTNAME_EXTERNAL via docker compose works correctly. 

## Expected behavior

{
    "QueueUrl": "http://test:4566/000000000000/test"
}

## Actual behavior

{
    "QueueUrl": "http://localhost:4566/000000000000/test"
}

# Steps to reproduce

## Command used to start LocalStack

```
version: '3.8'

services:
  localstack:
    container_name: "${LOCALSTACK_DOCKER_NAME-localstack_main}"
    image: localstack/localstack
    network_mode: bridge
    ports:
      - "4566:4566"
      - "${PORT_WEB_UI-8080}:${PORT_WEB_UI-8080}"
    environment:
      - SERVICES=sqs
      - DEBUG=${DEBUG- }
      - DATA_DIR=${DATA_DIR- }
      - PORT_WEB_UI=${PORT_WEB_UI- }
      - LAMBDA_EXECUTOR=${LAMBDA_EXECUTOR- }
      - KINESIS_ERROR_PROBABILITY=${KINESIS_ERROR_PROBABILITY- }
      - DOCKER_HOST=unix:///var/run/docker.sock
      - HOST_TMP_FOLDER=${TMPDIR}
    volumes:
      - "${TMPDIR:-/tmp/localstack}:/tmp/localstack"
      - "/var/run/docker.sock:/var/run/docker.sock"
```

`TMPDIR=/private$TMPDIR docker-compose up`

## Client code (AWS SDK code snippet, or sequence of "awslocal" commands)

`curl -v -d '{"variable":"HOSTNAME_EXTERNAL","value":"test"}' 'http://localhost:4566/?_config_'`

and then 

`aws --region=ap-southeast-2 --endpoint-url=http://localhost:4566 sqs create-queue --queue-name test`

**Comment by whummer** [comment]
Thanks for reporting @sougat818 . This should be fixed in #3579 - can you please pull the `latest` Docker image and give it another try? Please report here if the problem persists.

**Comment by sougat818** [comment]
Thanks @whummer & @ShadabShariff . I have verified that this is resolved on latest localstack & localstack-light docker images. Thanks a lot for the quick fix.
